gooddata-java-model/pom.xml

```diff
@@ -33,6 +33,10 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
```

Comment:
could we avoid adding dependencies to model? We have another issue asking to remove spring from model at all.

Reply:
ok, I will try to check if there's a possibility not to use this dependency

Reply:
Well, as this is implementation of `Page` interface which depends on other spring libraries (`UriComponentsBuilder` from `spring-web`), it will be probably better to keep this class in service. Closing PR